docker/createml.go

```diff
@@ -200,15 +200,54 @@ func PutManifestList(a *types.AuthInfo, yamlInput types.YAMLInput, ignoreMissing
 	}
 	defer resp.Body.Close()
 
+	var finalDigest string
 	if statusSuccess(resp.StatusCode) {
 		dgstHeader := resp.Header.Get("Docker-Content-Digest")
 		dgst, err := digest.Parse(dgstHeader)
 		if err != nil {
 			return "", err
 		}
-		return string(dgst), nil
+		finalDigest = string(dgst)
+	} else {
+		return "", fmt.Errorf("Registry push unsuccessful: response %d: %s", resp.StatusCode, resp.Status)
+	}
+	// if the YAML includes additional tags, push the added tag references. No other work
+	// should be required as we have already
```

Comment:
Building up interest sometimes takes introducing a bit of PR-style mystery around things ;) =) 

Real story: saw that I forgot to type the rest, got hungry for lunch and decided I could take care of pushing an update later :)